### PR TITLE
Add navigation tree view, add ui view

### PR DESCRIPTION
**Pull request by zirka28**

### CODE REVIEW

File: moninag/moninag/urls.py [hunk 1]
```diff
@@ -23,6 +24,7 @@
     url(r'^api/1/profile/', include('user_profile.urls')),
     url(r'^api/1/server/', include('server.urls')),
     url(r'^api/1/service/', include('service.urls')),
+    url(r'^api/1/tree/', get_navigation_tree),
     url(r'^auth/', include('registration.urls')),
     url(r'^', include('home.urls')),
-]
+]
```
Comment (hutsulyk) [inline moninag/moninag/urls.py]: Endline

File: moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        this.router.navigate(['/services', this.selectedService.id]);
+        event.stopPropagation();
+    }
+
+    gotoCheck(check:Check): void {
+        this.selectedCheck = check;
+        this.router.navigate(['checks', this.selectedCheck.id]);
+        event.stopPropagation();
+    }
+}
```
Comment (ghost) [inline moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts:53]: endline?

File: moninag/utils/navigation_tree_view.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
+from django.http import JsonResponse
+from server.models import Server
```
Comment (ghost) [inline moninag/utils/navigation_tree_view.py]: group imports

File: moninag/utils/navigation_tree_view.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+from django.http import JsonResponse
```
Comment (ghost) [inline moninag/utils/navigation_tree_view.py:3]: module docstring

File: moninag/utils/navigation_tree_view.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+from server.models import Server
+from service.models import Service
+from check.models import Check
+
+
+def get_navigation_tree(request):
```
Comment (ghost) [inline moninag/utils/navigation_tree_view.py]: method docstring

File: moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts (hunk 2 above)
Comment (hutsulyk) [inline moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts:53]: Endline :slightly_smiling_face:

**Review by ghost — CHANGES_REQUESTED** [review]

File: moninag/static/src/ts/app/navs/side-nav/side-nav.service.ts [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+import { Check } from '../../checks/check';
+import { Plugin } from '../../checks/plugin';
+import { Service } from '../../services/services';
+import { Server } from '../../servers/model';
+
+
+
+@Injectable()
+
+export class SideNavService {
```
Comment (hutsulyk) [inline moninag/static/src/ts/app/navs/side-nav/side-nav.service.ts]: Delete empty line between class and `@Injectable()` :upside_down_face:

**Review by hutsulyk — COMMENTED** [review]
Interesting that in Firefox menu doesn't work. Firefox doesn't know `$event`. In Chrome it works :upside_down_face: 

Great job! Fix comments only :slightly_smiling_face:

File: moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts (hunk 2 above)
Comment (zirka28) [reply moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts:53]: done

Comment (zirka28) [reply moninag/static/src/ts/app/navs/side-nav/side-nav.component.ts:53]: done

File: moninag/utils/navigation_tree_view.py (hunk 4 above)
Comment (zirka28) [reply moninag/utils/navigation_tree_view.py:3]: done

**Review by hutsulyk — APPROVED** [review]

**Review by ghost — APPROVED** [review]
awesome work! :+1:

**Pull request merged**